activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeTaskAssigneeTest.java

```diff
@@ -228,5 +229,73 @@ public void createStandaloneTaskForUsersAndAdminReassignUser() {
                                               .build());
     }
 
+    @Test
+    public void userCanReassignClaimedTaskToCandidateUsers() {
+
+        String taskId = createTask("garth");
+        //assign the new task
+        userAssignTask(taskId,"garth", "dean");
+        //reassign task to candidate
+        userAssignTask(taskId,"dean", "garth");
+
+        test_shouldReturnIllegalStateExceptionWhenUserIsNotAssigneeOrOwner(taskId,"dean", "garth");
+
+        cleanUp(taskId);
+    }
+    
+    private String createTask(String user){
+        securityUtil.logInAs(user);
+
+        taskRuntime.create(TaskPayloadBuilder.create()
+                .withName("group task")
+                .withCandidateUsers("dean")
+                .withCandidateUsers("garth")
+                .build());
+
+        // the owner should be able to see the created task
+        Page<Task> tasks = taskRuntime.tasks(Pageable.of(0,
+                50));
 
+        assertThat(tasks.getContent()).hasSize(1);
+
+        Task task = tasks.getContent().get(0);
+        assertThat(task.getAssignee()).isNull();
+        assertThat(task.getStatus()).isEqualTo(Task.TaskStatus.CREATED);
+        return task.getId();
```

Comment:
I don't think you need these lines here. As I'm expecting this to be already covered by another test. You can return directly the result of `taskRuntime.create`.